src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -913,5 +913,14 @@ See the accompanying LICENSE file for applicable license.
       </xsl:variable>
       <xsl:sequence select="$topicType"/>
     </xsl:function>
+    
+    <xsl:function name="dita-ot:insideExcludedDraftElement" as="xs:boolean">
```

Comment:
Because all uses of `dita-ot:insideExcludedDraftElement` are negated with `not()`, I think for readability this should be inverted so that we don't need to always use `not()` on the result. However, I can't come up with a good name for it.

Reply:
dita-ot:notInsideExcludedDraftElement() ?